sapphire/analysis/reconstructions.py

```diff
@@ -447,3 +441,17 @@ def _store_reconstruction(self, coincidence, core_x, core_y, theta, phi,
             row['s%d' % number] = True
 
         row.append()
+
+    def _get_active_stations(self):
+        """Return station numbers with non-empty event table in datafile"""
+        def stations_with_events():
```

Comment:
The generator is faster because it calls a function that _extends_ the list outside of the VM (only using the Python VM for yield), while the append version _appends_ an item N-times while iterating inside the VM by setting up and calling list.append() from the VM N-times.

Speed difference is unimportant for this case where N=125